siac/hostcmd.go

```diff
@@ -51,6 +52,61 @@ Doing so will override the standard connectivity checks.`,
 	}
 )
 
+// parseSize converts strings of form 10GB to a size in bytes
+// If parsing fails the original value is returned unmodified
+func parseSize(value string) string {
+	var suffix string
+	var size int
```

Comment:
I didn't want the variables to get between the comment and the if statement so I placed it above, if you feel it's better the other way I can move it.